### PR TITLE
Fix search request & response

### PR DESCRIPTION
**Pull request by Mpdreamz**
- validate response
- Add new aggregations and fix new tests failures for request

### CODE REVIEW

File: specification/specs/common.ts [hunk 1]
```diff
@@ -99,8 +101,8 @@ type ScrollIds = string // TODO: array of ScrollIds
 type CategoryId = string
 type ActionIds = string // TODO: check if this should be an array of ActionId
 
-type Id = string
-type Ids = string | string[]
+type Id = string | long
```
Comment (delvedor) [inline specification/specs/common.ts:104]: This does not feel right, can you point me to the yaml test that is failing?

Comment (Mpdreamz) [reply specification/specs/common.ts:104]: the `ids_query` accepts both

Comment (delvedor) [reply specification/specs/common.ts:104]: Then we should add this edge case only there, every other API needs a string and never a number :)

Comment (philkra) [reply specification/specs/common.ts:104]: I actually would vote to not have a union for `Id` it's a string everywhere else

Comment (Mpdreamz) [reply specification/specs/common.ts]: updated, it was in two places (pinned_query and LikeDocument).

**Review by philkra — APPROVED** [review]
lgtm
